<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LTPCHE102076955" name="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest" tests="17" failures="4" timestamp="2021-01-03T08:46:25 IST" time="2384.251" errors="1">
  <testcase name="deleteAllData_GrowthLogPageTest" time="40.515" classname="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest"/>
  <system-out/>
  <testcase name="verifyGrowthLogLinkIsDisplayed_GrowthLogTest" time="110.216" classname="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest"/>
  <system-out/>
  <testcase name="verifyNavigationToGrowthLogPage_GrowthLogTest" time="55.546" classname="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest"/>
  <system-out/>
  <testcase name="verifyAddMomentSectionIsDisplayed_GrowthLogTest" time="61.089" classname="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest"/>
  <system-out/>
  <testcase name="verifyNavgationToAddNewMomentScreenFromGrowthLog_GrowthLogTest" time="60.876" classname="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest"/>
  <system-out/>
  <testcase name="verifyMomentCreationWithImageMilestoneNoteHeightWeight_GrowthLogPageTest" time="189.080" classname="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest"/>
  <system-out/>
  <testcase name="verifyBackButton_GrowthLogPageTest" time="62.815" classname="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest"/>
  <system-out/>
  <testcase name="verifyEditMomentCardWithoutImage_GrowthLogPageTest" time="138.428" classname="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest"/>
  <system-out/>
  <testcase name="verifyMomentShareExternalAppsWithoutImage_GrowthLogPageTest" time="172.731" classname="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Check if Messages app is displayed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Check if Messages app is displayed expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest.verifyMomentShareExternalAppsWithoutImage_GrowthLogPageTest(GrowthLogPageTest.java:407)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyMomentShareExternalAppsWithoutImage_GrowthLogPageTest -->
  <system-out/>
  <testcase name="verifyDeleteMomentCardWithoutImage_GrowthLogPageTest" time="197.080" classname="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Check if Moment deleted from Growth log page-Check for Note displayed expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Check if Moment deleted from Growth log page-Check for Note displayed expected [false] but found [true]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest.verifyDeleteMomentCardWithoutImage_GrowthLogPageTest(GrowthLogPageTest.java:572)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyDeleteMomentCardWithoutImage_GrowthLogPageTest -->
  <system-out/>
  <testcase name="verifyMomentWithoutImageShareInGroupChat_GrowthLogPageTest" time="124.056" classname="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest"/>
  <system-out/>
  <testcase name="verifyFullPageImageView_GrowthLogPageTest" time="132.636" classname="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.pom.crimson.listeners.LogAssert.onAssertFailure(LogAssert.java:41)
at org.testng.asserts.SoftAssert.doAssert(SoftAssert.java:23)
at org.testng.asserts.Assertion.assertNotNull(Assertion.java:364)
at com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest.verifyFullPageImageView_GrowthLogPageTest(GrowthLogPageTest.java:628)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- verifyFullPageImageView_GrowthLogPageTest -->
  <system-out/>
  <testcase name="verifyMoreButtonFullPageImageView_GrowthLogPageTest" time="167.122" classname="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest"/>
  <system-out/>
  <testcase name="verifyCrossBtnFullImageView_GrowthLogPageTest" time="164.545" classname="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest"/>
  <system-out/>
  <testcase name="verifyEditMomentFullImageView_GrowthLogPageTest" time="278.200" classname="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Check if Weight saved in backend expected [99.71] but found [73.47],
	Check if Height saved in backend expected [13.92] but found [5.69],
	Check if Note saved in backend expected [nBXc5QYavlwpM2PNIKwM] but found [1ugK3JPvQvUPXzoAmegC],
	Check if Note saved in backend expected [nBXc5QYavlwpM2PNIKwM1234] but found [1ugK3JPvQvUPXzoAmegC]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Check if Weight saved in backend expected [99.71] but found [73.47],
Check if Height saved in backend expected [13.92] but found [5.69],
Check if Note saved in backend expected [nBXc5QYavlwpM2PNIKwM] but found [1ugK3JPvQvUPXzoAmegC],
Check if Note saved in backend expected [nBXc5QYavlwpM2PNIKwM1234] but found [1ugK3JPvQvUPXzoAmegC]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest.verifyEditMomentFullImageView_GrowthLogPageTest(GrowthLogPageTest.java:867)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyEditMomentFullImageView_GrowthLogPageTest -->
  <system-out/>
  <testcase name="verifyShareInGroupChatFullImageView_GrowthLogPageTest" time="190.520" classname="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest"/>
  <system-out/>
  <testcase name="verifyShareInExternalAppsFullImageView_GrowthLogPageTest" time="238.796" classname="com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Check if Weight saved in backend expected [59.89] but found [92.75],
	Check if Height saved in backend expected [29.69] but found [30.52],
	Check if Note saved in backend expected [wU2ZlNmJaOj7Pcan57z3] but found [H8bWBV2DXwGOhzST3rq5],
	Check if Messages app displayed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Check if Weight saved in backend expected [59.89] but found [92.75],
Check if Height saved in backend expected [29.69] but found [30.52],
Check if Note saved in backend expected [wU2ZlNmJaOj7Pcan57z3] but found [H8bWBV2DXwGOhzST3rq5],
Check if Messages app displayed expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest.verifyShareInExternalAppsFullImageView_GrowthLogPageTest(GrowthLogPageTest.java:1052)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyShareInExternalAppsFullImageView_GrowthLogPageTest -->
  <system-out/>
</testsuite> <!-- com.pom.crimson.testcases.momentlog.smoke.GrowthLogPageTest -->
